packages/EasyErrorHandler/src/Bridge/Symfony/DependencyInjection/Configuration.php

```diff
@@ -15,6 +15,7 @@ public function getConfigTreeBuilder(): TreeBuilder
 
         $treeBuilder->getRootNode()
             ->children()
+                ->booleanNode('api_platform')->defaultTrue()->end()
```

Comment:
@itorgov ok so if the intention of this config/param is to allow devs to keep using the default api platform listener, let's make it more explicit and call it something like `override_api_platform_listener` 😉

Reply:
Done.